enderio-base/src/main/java/com/enderio/base/mixin/client/ItemRendererMixin.java

```diff
@@ -14,12 +17,17 @@
 // TODO: We'll get rid of this mixin one day, waiting on a forge feature.
```

Comment:
It might be worth trying to work out how to integrate this into forge? I'd love to be rid of mixins haha

Reply:
That is also fair, more of an observation for a potential future :)